common/scripted_buttons/imperia_frankfurt_parliament_buttons.txt

```diff
@@ -0,0 +1,1885 @@
+﻿je_frankfurt_parliament_join_parliament_button = {
+	name = "je_frankfurt_parliament_join_parliament_button"
+	desc = "je_frankfurt_parliament_join_parliament_button_desc"
+	visible = {
+		has_global_variable = frankfurt_parliament_current_vote
+		global_var:frankfurt_parliament_current_vote = 0
+	}
+	ai_chance = {
+		base = 100
+	}
+	possible = {
+		custom_tooltip = {
+			text = frankfurt_parliament_joining_parliament
+			global_var:frankfurt_parliament_current_vote = 0
+		}
+		hidden_trigger = {
+			NOT = {
+				has_variable = frankfurt_parliament_joined_parliament
+			}
+		}
+	}
+	selected = {
+		has_variable = frankfurt_parliament_joined_parliament
+	}
+	effect = {
+		custom_tooltip = {
+			text = frankfurt_joins_the_parliament_tt
+			set_variable = frankfurt_parliament_joined_parliament
+			remove_variable ?= left_frankfurt_parliament
+		}
+	}
+}
+
+je_frankfurt_parliament_choose_club_button = {
```

Comment:
Is this tested in multiplayer?

Reply:
Nope, I'll set up a playdate with Lou

Reply:
Alright, our date just ended :p It all seems to work in multiplayer. I also found 2 other errors, that I'll get around to fixing